xbmc/pvr/timers/PVRTimerInfoTag.h

```diff
@@ -120,13 +122,46 @@ namespace PVR
 
     bool IsRecording(void) const { return m_state == PVR_TIMER_STATE_RECORDING; }
 
+    /*!
+      * @brief Checks whether is timer has a timer type. Can be false if
+      *        no PVR client supporting timers is enabled.
```

Comment:
then where does the timer come from if none of the clients support timers?
why do we need the shared_ptr here, when this is supposed to be a 1 <-> 1 relation?